### PR TITLE
finalMicHello

### PR DESCRIPTION
**Pull request by obibaadoma**
# Final work

### CODE REVIEW

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @obibaadoma 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that linters are failing. please fix your linters and resubmit

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -21,13 +19,14 @@
...
             <h1 class="head">I'm Marianne <br/>
-            Glad to see you!</h1>
-       
+            Glad to see you!</h1> 
         <p class="desc">
-            I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            I’m a software developer! I can help you build a product, 
+            feature or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded, don’t 
+            hestiate to contact me.
```
Comment (Kamba56) [inline index.html]: - [x] Please use an indentation of two spaces in all your documents. This is a convention and also [best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

File: index.html [hunk 2]
```diff
@@ -56,9 +55,293 @@ <h2 class="connect">LET'S CONNECT</h2>
...
+                                    <li><button class="language-buttons" type="button">javaScript</button></li>
+                                </ul>
+                            
+                                      <div class="btn">
+                                        <button type="submit" class="see">See projects</button>
+                                      </div>
+                                    </ul>
+                        </div>
+                        
+                            </section>
```
Comment (Kamba56) [inline index.html]: - [x] Please wrap all the work cards in one section (work section) and use grid to lay out all the cards in the setion.

File: index.html [hunk 3]
```diff
@@ -56,9 +55,293 @@ <h2 class="connect">LET'S CONNECT</h2>
...
+            <li><button class="language-buttons" type="button">css</button></li>
+            <li><button class="language-buttons" type="button">javaScript</button></li>
+        </ul>
+    
+	          <div class="btn">
+	            <button type="submit" class="see">See projects</button>
+	          </div>
+            </ul>
+</div>
     </section>
```
Comment (Kamba56) [inline index.html]: - [x] Kindly edit the alignment of the list of technologies and the button of each project card in the work section. You are expected to stick to the design template as much as possible.
 ### Your design

### Figma template
f1


- [x] Kindly reduce the spacing between project cards to match the Figma template.
### YOur design

### Figma template
f2

File: index.html [hunk 4]
```diff
@@ -56,9 +55,293 @@ <h2 class="connect">LET'S CONNECT</h2>
...
+                        </div>
+                        
+                            </section>
+                            <section class="contact-me">
+                              <div class="head-section-co">
+                                <div id="head-container-co">
+                                  <h1 class="head-co">
+                                    About, <br />
+                                    Myself
+                                  </h1>
```
Comment (Kamba56) [inline index.html]: - [x] Kindly change the heading of this section to a `hx` tag from `<h2>` to `<h6>`. Your document should have only one `<h1>` tag. This is best SEO practice.

File: index.html [hunk 5]
```diff
@@ -56,9 +55,293 @@ <h2 class="connect">LET'S CONNECT</h2>
...
+                                    </li>
+                                    <li>
+                                      <a href="#" target="_blank">
+                                        <img src="images/medium.png" alt="image" />
+                                      </a>
+                                    </li>
+                                  </ul>
+                                  <div class="newmes-co">
+                                    <button type="submit" class="go-newmes">My resume</button>
+                                  </div>
```
Comment (Kamba56) [inline index.html]: - [x] Kindly remove the spaces around the about section and add space inside the section to match the figma template.

- [x] Also, please align your social media icons horizontally to match the design 
### Your design

guidelines.
### Figma template
f3

File: index.html [hunk 6]
```diff
@@ -56,9 +55,293 @@ <h2 class="connect">LET'S CONNECT</h2>
...
+                                        src="./images/Separator-Bottom.png"
+                                        alt=""
+                                        class="separator"
+                                      />
+                                      <div class="event18">
+                                        <h3>Skills</h3>
+                                        <button type="submit" class="dropbtn">
+                                          <img src="./images/Icon-Chevron-Right-Larger.png" alt="" />
+                                        </button>
+                                      </div>
```
Comment (Kamba56) [inline index.html]: - [x] As demanded by the project requirements, kindly make the list of skills categories to an unordered list. Your skills section should look like this.
```html
<ul>
  <li>
    Languages
    <ul>
      <li>HTML</li>
      ...
  </li>
  <li>Skills</li>
  <li>Frameworks</li>
</ul>
```


- [x] Kindly add the icons to the right of the categories and also the line under the Frameworks and skills categories.
### Your design

### Figma template
f4

File: index.html [hunk 7]
```diff
@@ -56,9 +55,293 @@ <h2 class="connect">LET'S CONNECT</h2>
...
+                                    id="detail"
+                                    placeholder="Write your message here"
+                                    required
+                                    maxlength="500"
+                                  ></textarea>
+                                  <button class="form-button go-newmes" type="submit">
+                                    Get in touch
+                                  </button>
+                                </form>
+                                ​
```
Comment (Kamba56) [inline index.html]: - [x] Kindly remove this part of the document for now. You can start it somewhere till you need it. There will be a separate project milestone for them.

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @obibaadoma, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] All linters passed

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**